src/components/TracingVisualization/TracingVisualization.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { IApolloServerTracing } from "@/types";
```

Comment:
There is an issue with Jest atm not recognising these imports. I am yet to investigate and fix, could we use `../../` for now please?

Reply:
Update. I have now fixed the alias imports in jest.